src/main/java/org/extism/chicory/sdk/ChicoryModule.java

```diff
@@ -3,23 +3,14 @@
 import com.dylibso.chicory.aot.AotMachine;
 import com.dylibso.chicory.runtime.Module;
 
-class ManifestModuleMapper {
-    private final Manifest manifest;
+public class ChicoryModule {
```

Comment:
Removed this `ManifestModuleMapper`, which has now become the `Linker` and moved some ancillary code inside this `ChicoryModule`